sample/src/main/java/com/airbnb/android/showkasesample/ShowkaseTheme.kt

```diff
@@ -5,10 +5,9 @@ import androidx.compose.material.MaterialTheme
 import androidx.compose.material.darkColors
 import androidx.compose.material.lightColors
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.res.colorResource
 
 @Composable
-fun ShowkaseTheme(children: @Composable()() -> Unit) {
```

Comment:
ha this rings a bell. Showkase was started when Compose was still in dev preview. At the time, this sytax used to be needed 😅

Reply:
Aha! That makes sense! 😄